HeterogeneousCore/AlpakaTest/plugins/alpaka/TestAlgo.dev.cc

```diff
@@ -176,6 +176,24 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
         output[i] = {x, input[i].y(), input[i].z(), input[i].id(), input[i].flags(), input[i].m()};
       }
     }
+
+    template <typename TAcc, typename = std::enable_if_t<alpaka::isAccelerator<TAcc>>>
+    ALPAKA_FN_ACC void operator()(TAcc const& acc,
```

Comment:
Yes, I agree it's simpler to use
```suggestion
    ALPAKA_FN_ACC void operator()(Acc1D const& acc,
```